src/api/libcellml/units.h

```diff
@@ -415,14 +422,7 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
 	*
 	* @return Either @c true or @c false, depending if the units were successfully updated.
 	*/
-    static bool Units::updateUnitMultiplier(double &multiplier, const UnitsPtr &units, double uExp, double logMult, int direction);
-
-private:
-    Units(); /**< Constructor */
-    explicit Units(const std::string &name); /**< Constructor with std::string parameter*/
-
-    struct UnitsImpl; /**< Forward declaration for pImpl idiom. */
-    UnitsImpl *mPimpl; /**< Private member to implementation pointer */
+    static bool updateUnitMultiplier(double &multiplier, const UnitsPtr &units, double uExp, double logMult, int direction);
```

Comment:
There are really no private methods this is because we use the Pimpl idiom.  Any private methods should be added as public methods of the private implementation.

Reply:
So we would just declare it inside of the `Units::UnitsImpl` struct? (In that case, would I keep the docstring as well or just declare the function inside?)

Reply:
If it doesn't require any access to the details of the units then just leave it as a function, documenting it is for other developers to understand the intentions of the function that should be done no matter where the function lives.